plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -152,7 +152,7 @@ func isVolumeConflict(volume v1.Volume, pod *v1.Pod) bool {
 // - Ceph RBD forbids if any two pods share at least same monitor, and match pool and image.
 // - ISCSI forbids if any two pods share at least same IQN, LUN and Target
 // TODO: migrate this into some per-volume specific code?
-func NoDiskConflict(pod *v1.Pod, meta interface{}, nodeInfo *schedulercache.NodeInfo) (bool, []algorithm.PredicateFailureReason, error) {
+func NoDiskConflict(pod *v1.Pod, meta algorithm.PredicateMetadata, nodeInfo *schedulercache.NodeInfo) (bool, []algorithm.PredicateFailureReason, error) {
```

Comment:
I think that this change should be moved to a separate PR.
If the preemption will contain serious bugs and we will have to revert it, we shouldn't be forced to also revert this cleanup.
Please move it to a separate PR.

Reply:
It isn't small - it's almost 200 lines. I still think we should separate it.

Reply:
At this point and given the extension we got for 1.8, a single PR will be easier to manage.

Reply:
I still don't agree with that :)